scheduler/tasks.py

```diff
@@ -188,3 +225,28 @@ def run(self, metric_name, metric_value, session=None, **kwargs):
 
 
 fire_metric = FireMetric()
+
+
+class RequeueFailedTasks(Task):
+
+    """
+    Task to requeue failed schedules.
+    """
+    name = "seed_scheduler.scheduler.tasks.requeue_failed_tasks"
+
+    def run(self, **kwargs):
+        """
+        Runs an instance of a scheduled task
+        """
+        l = self.get_logger(**kwargs)
+        failures = ScheduleFailure.objects.all()
+        l.info("Attempting to requeue <%s> failed schedules" %
+               failures.count())
+        for failure in failures:
+            schedule_id = str(failure.schedule_id)
+            # Cleanup the failure before requeueing it.
+            failure.delete()
+            deliver_task.delay(schedule_id)
```

Comment:
I haven't made re-queueing a scheduled task yet for this purpose. I guessed we would want to run this for a while and see what kind of failures land up here in this table and potential build tools around handling cases that are never going to work again.